assignments/array-methods.js

```diff
@@ -64,9 +64,9 @@ console.log(fullName);
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
-runners.forEach(function(names)
+allCaps = runners.map(function(names)
 {
-  allCaps.push(names.first_name.toUpperCase())
+  return names.first_name.toUpperCase()
```

Comment:
Well done! You're taking advantage of the fact that `map` creates a new array for you, and pushes elements into it based on the result of the callback. :100: